wp/lib/bap_wp/src/environment.mli

```diff
@@ -201,6 +212,27 @@ val is_x86 : Bap.Std.Arch.t -> bool
     when generating symbols in the function specs at a function call site. *)
 val use_input_regs : t -> bool
 
+(** Obtains a Z3 expression that checks if an address is within the stack region
+    of memory: i.e. `stack_min <= addr <= stack_max`. *)
```

Comment:
Ok here's the doc. Maybe be more explicit, e.g. "`in_stack env addr` is the constraint `STACK_MIN <= addr <= STACK_MAX` as defined by the `env`."